modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/definition/tree/manager/test/ObjectDefinitionTreeUtilTest.java

```diff
@@ -1081,6 +1083,81 @@ public void testBindPublishedObjectDefinitions() throws Exception {
 			_objectEntryLocalService, _objectRelationshipLocalService);
 	}
 
+	@Test
+	public void testDeleteNodeObjectEntry() throws Exception {
+
+		// Delete object entry
+
+		ObjectDefinition rootObjectDefinition1 =
+			_addAndPublishCustomObjectDefinition();
+		ObjectDefinition childObjectDefinition1 =
+			_addAndPublishCustomObjectDefinition();
+
+		ObjectRelationship objectRelationship1 =
+			ObjectRelationshipTestUtil.addObjectRelationship(
+				_objectRelationshipLocalService, rootObjectDefinition1,
+				childObjectDefinition1);
+
+		ObjectDefinition rootObjectDefinition2 =
+			_addAndPublishCustomObjectDefinition();
+
+		ObjectRelationship objectRelationship2 =
+			ObjectRelationshipTestUtil.addObjectRelationship(
+				_objectRelationshipLocalService, rootObjectDefinition2,
+				childObjectDefinition1);
+
+		TreeTestUtil.bind(
+			_objectRelationshipLocalService,
+			List.of(objectRelationship1, objectRelationship2));
+
+		DefaultObjectEntryManager defaultObjectEntryManager =
+			(DefaultObjectEntryManager)_objectEntryManager;
+
+		ObjectEntry rootObjectEntry1 = _addObjectEntry(
+			rootObjectDefinition1, Collections.emptyMap());
+
+		ObjectField objectRelationship1ObjectField2 =
+			_objectFieldLocalService.getObjectField(
+				objectRelationship1.getObjectFieldId2());
+
+		ObjectEntry serviceBuilderChildObjectEntry1 = _addObjectEntry(
+			childObjectDefinition1,
+			HashMapBuilder.<String, Serializable>put(
+				objectRelationship1ObjectField2.getName(),
+				rootObjectEntry1.getObjectEntryId()
+			).build());
+
+		AssertUtils.assertFailure(
+			UnsupportedOperationException.class, null,
+			() -> defaultObjectEntryManager.deleteObjectEntry(
+				childObjectDefinition1,
+				serviceBuilderChildObjectEntry1.getObjectEntryId()));
+
+		ObjectEntry serviceBuilderChildObjectEntry2 = _addObjectEntry(
```

Comment:
When you rename it, it will be clear that this entry won't be a child of anyone. It's just a simple entry of the object definition that CAN be a child in some root context (but it's not in this context).

Reply:
done